Power_Board_V.1 Run.cfg

```diff
@@ -0,0 +1,28 @@
+# This is an STM32F0DISCOVERY board with a single STM32F051R8Tx chip
+#
+# Generated by System Workbench for STM32
+# Take care that such file, as generated, may be overridden without any early notice. Please have a look to debug launch configuration setup(s)
+
+source [find interface/stlink.cfg] 
+
+set WORKAREASIZE 0x2000
+
+transport select "hla_swd"
+
+set CHIPNAME STM32F051R8Tx
+
+# Enable debug when in low power modes
+set ENABLE_LOW_POWER 1
+
+# Stop Watchdog counters when halt
+set STOP_WATCHDOG 1
+
+# STlink Debug clock frequency
+set CLOCK_FREQ 4000
+
+# use hardware reset, connect under reset
+# connect_assert_srst needed if low power mode application running (WFI...)
+reset_config srst_only srst_nogate connect_assert_srst
+set CONNECT_UNDER_RESET 1
+
+source [find target/stm32f0x.cfg]
```

Comment:
Hey Aman, this file is autogenerated, so it can be prone to cause merge conflicts. Would you be able to remove it, then we can merge this into master!